app/models/loan_response.rb

```diff
@@ -74,6 +74,10 @@ def blank?
       (embeddable_media.blank? || embeddable_media.url.blank?)
   end
 
+  def present?
+    !blank?
+  end
+
```

Comment:
I'm pretty sure this is provided by ActiveSupport. Is there a reason you're reimplementing it?